### PR TITLE
Fix promoteid style test

### PR DESCRIPTION
**Pull request by ank27**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### Summary of changes

PromoteId does not have a default value in source property. Removed the tests for default values.

### CODE REVIEW

**Comment by kiryldz** [comment]
Please mention this PR in associated stylegen PR.

**Review by kiryldz — APPROVED** [review]

**Review by alexander-kulikovskii — APPROVED** [review]

**Review by Chaoba — APPROVED** [review]

File: extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt [hunk 1]
```diff
@@ -764,24 +764,6 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
        */
       get() = StyleManager.getStyleSourcePropertyDefaultValue("geojson", "generateId").silentUnwrap()
 
-    /**
-     * A property to use as a feature id (for feature state). Either a property name, or
-     * an object of the form `{<sourceLayer>: <propertyName>}`.
-     */
-    val defaultPromoteId: PromoteId?
```
Comment (pengdev) [inline extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt:771]: not sure we should remove the default getter, could you double check the line-gradient default property getter? we have similar situation there.

Comment (ank27) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt:771]: default getters are used only for tests, so if tests are not present. we should remove it as well.

Comment (pengdev) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt:771]: These are public APIs 😄  So user could still decide to use them, if they are not used in our example app.

Comment (pengdev) [reply extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt:771]: Sorry, just checked line-gradient default property getter is also removed from the code, we are consistent.

**Review by pengdev — APPROVED** [review]

**Comment by pengdev** [comment]
LGTM, please merge after the `run-style-test` passes.

**Comment by kiryldz** [comment]
Added `skip-changelog` as this feature was just introduced in rc.9, so no need for changelog here

**Pull request merged**